src/ecs/commands/runCommandInContainer.ts

```diff
@@ -109,14 +114,29 @@ export async function runCommandInContainer(
         })
 
         result = 'Succeeded'
-    } catch (error) {
-        if (CancellationError.isUserCancelled(error)) {
+    } catch (err) {
+        const error = err as Error
+        if (CancellationError.isUserCancelled(err)) {
             return
         }
 
         result = 'Failed'
-        getLogger().error('ecs: Failed to execute command in container, %O', error)
-        showViewLogsMessage(localize('AWS.ecs.runCommandInContainer.error', 'Failed to execute command in container.'))
+        const failedMessage = localize('AWS.ecs.runCommandInContainer.error', 'Failed to execute command in container.')
+        const missingPermissions = localize(
+            'AWS.ecs.runCommandInContainer.missingPermissions',
+            'Execute Command not allowed. See documentation for required permissions.'
```

Comment:
Wording is kind of vague. _What_ doesn't have permissions?
If it's the user's role, perhaps phrase it by placing the subject first:
`Insufficient user role permissions for "${commandName}".  See documentation for required permissions.`

The user already knows what they were wanting to do. With errors, they need to know why they **could not** perform the action.